python/test/test_polars.py

```diff
@@ -30,6 +30,7 @@ def test_infer():
     assert dp.RuntimeType.infer(series) == "Series"
     assert dp.RuntimeType.infer(pl.DataFrame(series)) == "DataFrame"
     assert dp.RuntimeType.infer(pl.LazyFrame(series)) == "LazyFrame"
+    assert dp.RuntimeType.infer(pl.col("A")) == "Expr"
```

Comment:
As this gets longer, it feels more like a test that should be parameterized:
```
@pytest.mark.parametrize("expected_type,obj", {
    'Series': series,
    'DataFrame': pl.DataFrame(series),
    'LazyFrame': pl.LazyFrame(series),
    'Expr': pl.col("A")
}.items())
```

Reply:
I don't think works with optional pl?

Reply:
good point -- could wrap this in a class to group all the tests that need the `pl`, but not worth it.